libnmstate/nmstate.py

```diff
@@ -166,6 +166,10 @@ def _get_interface_info_from_plugins(plugins):
             iface[IFACE_PLUGIN_SRC_METADATA] = [plugin.name]
             iface_name = iface[Interface.NAME]
             iface_type = iface.get(Interface.TYPE, InterfaceType.UNKNOWN)
+            # Veth interface is not allowed to have duplicate names with
+            # ethernet interfaces.
```

Comment:
This comment does not explain why you are changing the type from veth to ethernet.

Reply:
Modified, the new comment is: 

```
# When NetworkManager does not support Veth interfaces will report  
# them as Ethernet. Therefore, the information will not be merged   
# correctly from Nispor and NetworkManager, in order to solve that, 
# we are considering Veth as Ethernet when merging.  
```